make-images.ipynb

```diff
@@ -11,42 +11,169 @@
+   "execution_count": 4,
+   "id": "7d8cfd4c-913e-4983-92ef-c40567421331",
+   "metadata": {
+    "jupyter": {
+     "source_hidden": true
+    }
+   },
+   "outputs": [
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "0036ae1165254d019e392a3ebd9b7905",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "VBox(children=(Text(value='', description='OSM username:', placeholder='Steve'), Text(value='', description='O…"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import ipywidgets as widgets\n",
+    "\n",
+    "osm_user = widgets.Text(\n",
+    "    placeholder='Steve',\n",
+    "    description='OSM username:',\n",
+    "    disabled=False   \n",
+    ")\n",
+    "\n",
+    "osm_password = widgets.Text(\n",
+    "    placeholder='your password',\n",
+    "    description='OSM password:',\n",
+    "    disabled=False   \n",
+    ")\n",
+    "\n",
+    "file_url = widgets.Text(\n",
+    "    placeholder='https://osm-internal.download.geofabrik.de/south-america/argentina-internal.osh.pbf',\n",
+    "    description='Link to OSM history file:',\n",
+    "    disabled=False   \n",
+    ")\n",
+    "widgets.VBox([\n",
+    "    osm_user,\n",
+    "    osm_password,\n",
+    "    file_url\n",
+    "    ])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "15b753ea-4489-4620-9c19-e46e59f0afeb",
+   "metadata": {},
+   "source": [
+    "Once you have added the details above, you can download your file by running the cell below this one:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "id": "582dbd2b-bd06-4dcf-934a-112c1de79fbf",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "--2024-12-20 21:19:10--  https://osm-internal.download.geofabrik.de/south-america/uruguay-internal.osh.pbf\n",
+      "Resolving osm-internal.download.geofabrik.de (osm-internal.download.geofabrik.de)... 65.109.22.172, 2a01:4f9:5a:1426::2\n",
+      "connected. to osm-internal.download.geofabrik.de (osm-internal.download.geofabrik.de)|65.109.22.172|:443... \n",
+      "HTTP request sent, awaiting response... 200 OK\n",
+      "Length: 87939811 (84M) [application/octet-stream]\n",
+      "Saving to: ‘uruguay-internal.osh.pbf’\n",
+      "\n",
+      "uruguay-internal.os 100%[===================>]  83.87M   190KB/s    in 3m 3s   \n",
+      "\n",
+      "2024-12-20 21:22:15 (468 KB/s) - ‘uruguay-internal.osh.pbf’ saved [87939811/87939811]\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "!python sendfile_osm_oauth_protector/oauth_cookie_client.py -u \"{osm_user.value}\" -p \"{osm_password.value}\" -c \"https://osm-internal.download.geofabrik.de/get_cookie\" -f netscape -o geofabrik_cookie.txt;wget --max-redirect 0 --load-cookies geofabrik_cookie.txt {file_url.value}"
```

Comment:
```suggestion
    "geofabrik_cookie = ('gf_download_oauth', widgets.Text(\n",
    "    placeholder='login|2018-04-12|WVRced34c...',\n",
    "    description='gf_download_oauth cookie value:',\n",
    "    disabled=False   \n",
    "))\n",
    "file_url = widgets.Text(\n",
    "    placeholder='https://osm-internal.download.geofabrik.de/south-america/argentina-internal.osh.pbf',\n",
    "    description='Link to OSM history file:',\n",
    "    disabled=False   \n",
    ")\n",
    "\n",
    "widgets.VBox([\n",
    "    geofabrik_cookie[1],\n",
    "    file_url\n",
    "    ])"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "15b753ea-4489-4620-9c19-e46e59f0afeb",
   "metadata": {},
   "source": [
    "Once you have added the details above, you can download your file by running the cell below this one:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "id": "582dbd2b-bd06-4dcf-934a-112c1de79fbf",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--2024-12-20 21:19:10--  https://osm-internal.download.geofabrik.de/south-america/uruguay-internal.osh.pbf\n",
      "Resolving osm-internal.download.geofabrik.de (osm-internal.download.geofabrik.de)... 65.109.22.172, 2a01:4f9:5a:1426::2\n",
      "connected. to osm-internal.download.geofabrik.de (osm-internal.download.geofabrik.de)|65.109.22.172|:443... \n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 87939811 (84M) [application/octet-stream]\n",
      "Saving to: ‘uruguay-internal.osh.pbf’\n",
      "\n",
      "uruguay-internal.os 100%[===================>]  83.87M   190KB/s    in 3m 3s   \n",
      "\n",
      "2024-12-20 21:22:15 (468 KB/s) - ‘uruguay-internal.osh.pbf’ saved [87939811/87939811]\n",
      "\n"
     ]
    }
   ],
   "source": [
    "cookie_name = geofabrik_cookie[0]\n",
    cookie_value = geofabrik_cookie[1].value\n",
    "\n",
    "!wget --max-redirect 0 --header="Cookie: {cookie_name}={cookie_value}" {file_url.value}"
```

Reply:
@gedankenstuecke Oh yeah, this diff is a nightmare. Anyway, I tried to apply it.

This replaces OSM login data with the Geofabrik cookie value. I've tested it and it works!

Probably some instructions would be needed in the notebook about getting the cookie value.

Reply:
Thanks for giving it a try! Do you want to just send over your version of the notebook (or make a PR to my branch) using the whole file? That still gives ugly diffs but at least sorts out any potential issues that could come from suggesting edits! 

Also I'd suggest maybe keeping the username/password option as well, so that people can decide whether they prefer pasting the cookie or using the exchange function to get the cookie, what do you think? 

My fear is that getting the cookie out of the browser maybe isn't super trivial, and so having the option for those comfortable with that approach would be good?

Reply:
> Do you want to just send over your version of the notebook (or make a PR to my branch) using the whole file?

> I'd suggest maybe keeping the username/password option as well

@gedankenstuecke I'll try keeping this option and submit a PR as soon as https://github.com/microsoft/vscode-jupyter/issues/16341 is fixed! Until then, here is the whole file in the same state as my suggestion above:

```
{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "1e92ba34-4a61-468f-b4da-ebb32432202d",
   "metadata": {},
   "source": [
    "# Create a before/after-map of OpenStreetMap data\n",
    "\n",
    "This notebook allows you to make a set of images of OSM data of an area from 2 dates, showing what was changed, making use of the [osm-mapping-party-before-after](https://github.com/amandasaurus/osm-mapping-party-before-after/) tool.\n",
    "\n",
    "## Getting started\n",
    "\n",
    "### If this is your first time using a notebook like this: \n",
    "\n",
    "1. You can use the up/down arrow keys of your keyboard to move between the different \"cells\" or regions of this notebook. The currently selected one will be highlighed on the left with a colour bar.\n",
    "2. You can \"run\" each of these cells/regions by either pressing the \"play\" button in the menu above, or by pressing CTRL+ENTER on your keyboard.\n",
    "3. If you are in the \"editing\" mode where you type into a cell, pressing ESC will take you back to the mode where the arrow keys work. \n",
    "\n",
    "### Uploading the OSM history file\n",
    "\n",
    "Before we can do the rest of the steps in this notebook, you will have to upload an OSM history file that covers your region of interest. You can find the corresponding files on [Geofabrik's internal download server](https://osm-internal.download.geofabrik.de/?landing_page=true). Login with your OSM account to get to the `internal` files, and then find the page for the your region of interest. Ideally, find the smallest region that covers your area of interest to minimize the filesize you need to upload. \n",
    "\n",
    "**The right file**: When you've picked your region, you will likely see two similarly named files listed: `regionname-internal.osh.pbf` and `regionname-latest-internal.osm.pbf`. The file you will want to download is the `regionname-internal.osh.pbf` (i.e. the one that includes `osh` and **no `latest`**).\n",
    "\n",
    "Once you have downloaded your file onto your computer, you need to upload it into your container. There are a few options that you can consider, depending on how you are using this notebook:\n",
    "\n",
    "#### Uploading a file you already have downloaded onto your computer\n",
    "\n",
    "* If you've launched this notebook in _MyBinder_, you can then open up the filebrowser on the top of the left hand side (The _file_ icon, or by pressing CTRL+SHIFT+F). \n",
    "* If you've launched this notebook locally, you're likely in the _notebook_ interface, switch back to the _tree_ view and upload your file there (or put it into the shared folder which is accessible by both the container and your normal operating system).\n",
    "\n",
    "#### Downloading a file directly from Geofabrik \n",
    "\n",
    "This option is particularly useful if you use this notebook on MyBinder, as it allows you to directly download the OSM history file into your Binder, without having to first download it onto your computer and then re-uploading it to the Binder, speeding things up considerably. As the OSM history files are only accessible with an OSM account, you will unfortunately have to fill out the form below, including your OSM username & password. \n",
    "\n",
    "**Note:** Your username & password will only be used to get a cookie that allows downloading the files and won't be permanently stored (and are not accessible to anyone else). If you are using _MyBinder_, your virtual machine will automatically shut down after more than 10 minutes of inactivity, destroying all data within the virtual machine in the process. If you are running this in a Docker container on your own computer the container will also destroy itself after shutting it down. \n",
    "\n",
    "If you decide to use the direct download from Geofabrik, enter the necessary details here: "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "7d8cfd4c-913e-4983-92ef-c40567421331",
   "metadata": {
    "jupyter": {
     "source_hidden": true
    }
   },
   "outputs": [
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "0036ae1165254d019e392a3ebd9b7905",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "VBox(children=(Text(value='', description='OSM username:', placeholder='Steve'), Text(value='', description='O…"
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import ipywidgets as widgets\n",
    "\n",
    "geofabrik_cookie = ('gf_download_oauth', widgets.Text(\n",
    "    placeholder='login|2018-04-12|WVRced34c...',\n",
    "    description='gf_download_oauth cookie value:',\n",
    "    disabled=False   \n",
    "))\n",
    "file_url = widgets.Text(\n",
    "    placeholder='https://osm-internal.download.geofabrik.de/south-america/argentina-internal.osh.pbf',\n",
    "    description='Link to OSM history file:',\n",
    "    disabled=False   \n",
    ")\n",
    "\n",
    "widgets.VBox([\n",
    "    geofabrik_cookie[1],\n",
    "    file_url\n",
    "    ])"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "15b753ea-4489-4620-9c19-e46e59f0afeb",
   "metadata": {},
   "source": [
    "Once you have added the details above, you can download your file by running the cell below this one:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "id": "582dbd2b-bd06-4dcf-934a-112c1de79fbf",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--2024-12-20 21:19:10--  https://osm-internal.download.geofabrik.de/south-america/uruguay-internal.osh.pbf\n",
      "Resolving osm-internal.download.geofabrik.de (osm-internal.download.geofabrik.de)... 65.109.22.172, 2a01:4f9:5a:1426::2\n",
      "connected. to osm-internal.download.geofabrik.de (osm-internal.download.geofabrik.de)|65.109.22.172|:443... \n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 87939811 (84M) [application/octet-stream]\n",
      "Saving to: ‘uruguay-internal.osh.pbf’\n",
      "\n",
      "uruguay-internal.os 100%[===================>]  83.87M   190KB/s    in 3m 3s   \n",
      "\n",
      "2024-12-20 21:22:15 (468 KB/s) - ‘uruguay-internal.osh.pbf’ saved [87939811/87939811]\n",
      "\n"
     ]
    }
   ],
   "source": [
    "cookie_name = geofabrik_cookie[0]\n",
    cookie_value = geofabrik_cookie[1].value\n",
    "\n",
    "!wget --max-redirect 0 --header="Cookie: {cookie_name}={cookie_value}" {file_url.value}"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "bfe0b5b8-cebd-4975-8447-f2189459dbcb",
   "metadata": {},
   "source": [
    "Once the file is completely downloaded (or if you uploaded the file using one of the other methods), you can continue with the next step below. \n",
    "\n",
    "## Starting to make the comparison images. \n",
    "\n",
    "When you run the code-cell below, it will generate some \"forms\" that will feel familiar form other web-forms, which you can use to enter all the necessary parameters to create your images. These forms allow you to add the _history file_ you have just uploaded, the two time-points you want to compare, the bounding box (i.e. the area you're interested in, which you can find e.g. [via bboxfinder](https://bboxfinder.com)) and the minimum/maximum zoom levels, if you want to create images at different zoom levels. \n",
    "\n",
    "Now, run the cell below and then enter all the necessary information in the forms:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "cf6cc023-ea39-46eb-885e-86a184dc1850",
   "metadata": {
    "jupyter": {
     "source_hidden": true
    }
   },
   "outputs": [
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "ef5e2a5f18f54653bf1e7d39e90559ac",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "VBox(children=(Dropdown(description='File:', options=('external/argentina-internal.osh.pbf', 'external/argenti…"
      ]
     },
     "execution_count": 1,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import ipywidgets as widgets\n",
    "import glob\n",
    "\n",
    "filename = widgets.Dropdown(\n",
    "    options=glob.glob(\"**/*.pbf\",recursive=True),\n",
    "    description='File:',\n",
    "    disabled=False,\n",
    ")\n",
    "\n",
    "bbox_label= widgets.Label(\n",
    "    value=\"BBox is in long/lat and can be found via bboxfinder.com\",\n",
    "    layout=widgets.Layout(width='100%')\n",
    ")\n",
    "\n",
    "bbox = widgets.Text(\n",
    "    placeholder='-61.975228,-33.749067,-61.953167,-33.733014',\n",
    "    description='BBox:',\n",
    "    disabled=False   \n",
    ")\n",
    "\n",
    "\n",
    "start_date = widgets.NaiveDatetimePicker(description='Start datetime')\n",
    "end_date = widgets.NaiveDatetimePicker(description='End datetime')\n",
    "\n",
    "date_label= widgets.Label(\n",
    "    value=\"You need to pick a time for the script to work, even if it's just 00:00\",\n",
    "    layout=widgets.Layout(width='100%')\n",
    ")\n",
    "\n",
    "min_zoom_level = widgets.Dropdown(\n",
    "    options=range(0,21),\n",
    "    description='Min Zoom:',\n",
    "    disabled=False,\n",
    ")\n",
    "\n",
    "max_zoom_level = widgets.Dropdown(\n",
    "    options=range(0,21),\n",
    "    description='Max Zoom:',\n",
    "    disabled=False,\n",
    ")\n",
    "\n",
    "num_frames = widgets.BoundedIntText(\n",
    "    min=2,\n",
    "    step=1,\n",
    "    value=2,\n",
    "    description='Number of frames:',\n",
    "    disabled=False\n",
    ")\n",
    "\n",
    "widgets.VBox([\n",
    "    filename,\n",
    "    bbox,\n",
    "    bbox_label,\n",
    "    start_date,\n",
    "    end_date,\n",
    "    date_label,\n",
    "    min_zoom_level,\n",
    "    max_zoom_level,\n",
    "    num_frames])"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "0fa54f42-e3f7-428b-b604-9450f4c659d7",
   "metadata": {},
   "source": [
    "Once you have set your parameters, you can run the cell below in the same way that you ran the cell above for creating the forms. \n",
    "\n",
    "Running the cell below will take a while, in particular if it's the first time you execute it, as it needs to download some external resources first. Subsequent runs will be faster. You can re-run it as many times as you want, and change any of the parameters using the forms above. \n",
    "\n",
    "Once you run the cell below, it will print a lot of output into this notebook, including some warnings. If everything works, the last things printed out will read *Generating comparison image for zoom XXX*, according to the zoom levels you picked. \n",
    "\n",
    "The resulting images will be saved as \"progress.filename.*parameters*.png/gif."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "d5932adf-763e-4a3a-8f96-9247cb90ecc2",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Extracting the OSM history for just this bounding box -61.975228,-33.749067,-61.953167,-33.733014\n",
      "[======================================================================] 100% \n",
      "Extracting 'before' data...\n",
      "[======================================================================] 100% \n",
      "Extracting 'after' data...\n",
      "[======================================================================] 100% \n",
      "Installing carto into /home/postgres/openstreetmap-carto/node_modules with npm...\n",
      "Wrote to /home/postgres/openstreetmap-carto/package.json:\n",
      "\n",
      "{\n",
      "  \"name\": \"openstreetmap-carto\",\n",
      "  \"version\": \"1.0.0\",\n",
      "  \"description\": \"![screenshot](https://raw.github.com/gravitystorm/openstreetmap-carto/master/preview.png)\",\n",
      "  \"main\": \"index.js\",\n",
      "  \"scripts\": {\n",
      "    \"test\": \"echo \\\"Error: no test specified\\\" && exit 1\"\n",
      "  },\n",
      "  \"keywords\": [],\n",
      "  \"author\": \"\",\n",
      "  \"license\": \"ISC\"\n",
      "}\n",
      "\n",
      "\n",
      "\u001b[K\u001b[?25h              \u001b[27m] | reify:lodash: \u001b[32;40mhttp\u001b[0m \u001b[35mfetch\u001b[0m GET 200 https://registry.npmjs.\u001b[0m\u001b[K.o\u001b[0m\u001b[K\n",
      "added 64 packages, and audited 65 packages in 5s\n",
      "\n",
      "1 package is looking for funding\n",
      "  run `npm fund` for details\n",
      "\n",
      "\u001b[31m\u001b[1m5\u001b[22m\u001b[39m vulnerabilities (2 \u001b[33m\u001b[1mmoderate\u001b[22m\u001b[39m, 3 \u001b[31m\u001b[1mhigh\u001b[22m\u001b[39m)\n",
      "\n",
      "To address all issues, run:\n",
      "  npm audit fix\n",
      "\n",
      "Run `npm audit` for details.\n",
      "Warning: style/landcover.mss:605:4 line-offset is unstable. It may change in the future.\n",
      "Warning: style/landcover.mss:608:6 line-offset is unstable. It may change in the future.\n",
      "Warning: style/water-features.mss:112:6 line-offset is unstable. It may change in the future.\n",
      "Warning: style/water-features.mss:114:6 line-offset is unstable. It may change in the future.\n",
      "Warning: style/water-features.mss:99:4 line-offset is unstable. It may change in the future.\n",
      "Warning: style/water-features.mss:105:4 line-offset is unstable. It may change in the future.\n",
      "Warning: style/landcover.mss:821:6 line-offset is unstable. It may change in the future.\n",
      "Warning: style/landcover.mss:814:4 line-offset is unstable. It may change in the future.\n",
      "Warning: style/landcover.mss:919:6 line-offset is unstable. It may change in the future.\n",
      "Warning: style/landcover.mss:921:6 line-offset is unstable. It may change in the future.\n",
      "Warning: style/landcover.mss:903:4 line-offset is unstable. It may change in the future.\n",
      "Warning: style/landcover.mss:911:6 line-offset is unstable. It may change in the future.\n",
      "Warning: style/admin.mss:533:10 line-offset is unstable. It may change in the future.\n",
      "Warning: style/admin.mss:529:10 line-offset is unstable. It may change in the future.\n",
      "Warning: style/admin.mss:520:8 line-offset is unstable. It may change in the future.\n",
      "Warning: style/roads.mss:2783:6 text-min-distance is deprecated. It may be removed in the future.\n",
      "Warning: style/roads.mss:2751:6 text-min-distance is deprecated. It may be removed in the future.\n",
      "Warning: style/admin.mss:465:4 text-largest-bbox-only is experimental. It may change, be renamed or removed in the future.\n",
      "Warning: style/admin.mss:486:2 text-largest-bbox-only is experimental. It may change, be renamed or removed in the future.\n",
      "Warning: style/landcover.mss:566:29 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:566:29 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:565:29 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:565:29 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:607:6 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:607:6 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:563:8 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:563:8 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:506:29 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:506:29 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:505:29 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:505:29 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:493:27 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:493:27 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:492:27 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:492:27 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:456:8 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:456:8 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:441:8 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:441:8 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:423:6 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:423:6 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:377:8 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:377:8 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:364:29 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:364:29 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:363:29 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:363:29 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:361:10 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:361:10 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:321:10 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:321:10 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:260:27 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:259:27 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:233:8 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:233:8 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:201:29 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:201:29 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:200:29 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:200:29 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:699:25 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:698:25 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:695:6 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:695:6 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:689:25 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:688:25 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:685:6 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:685:6 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:676:8 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:676:8 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:674:27 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:673:27 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:660:27 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:659:27 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:652:25 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:651:25 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:645:25 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:645:25 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:644:25 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:644:25 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:635:25 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:634:25 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:623:27 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:623:27 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:622:27 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:622:27 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:619:8 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:619:8 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:601:4 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:585:27 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:584:27 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:569:27 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:568:27 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:552:27 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:551:27 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:544:25 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:543:25 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:538:25 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:537:25 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:532:25 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:531:25 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:526:25 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:525:25 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:520:25 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:519:25 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:514:25 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:513:25 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:504:8 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:504:8 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:502:27 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:501:27 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:491:6 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:491:6 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:489:25 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:488:25 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:483:25 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:482:25 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:476:27 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:475:27 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:468:27 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:467:27 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:460:25 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:459:25 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:453:6 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:453:6 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:451:19 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:451:19 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:450:19 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:450:19 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:445:25 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:445:25 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:444:25 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:444:25 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:438:6 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:438:6 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:435:6 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:435:6 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:427:25 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:426:25 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:415:8 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:415:8 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:401:8 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:401:8 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:388:6 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:388:6 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:381:25 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:380:25 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:374:6 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:374:6 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:372:19 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:372:19 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:371:19 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:371:19 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:358:8 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:358:8 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:356:21 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:356:21 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:355:21 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:355:21 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:347:25 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:346:25 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:337:27 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:336:27 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:333:8 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:333:8 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:325:25 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:324:25 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:318:8 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:318:8 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:311:27 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:310:27 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:301:8 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:301:8 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:295:27 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:295:27 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:294:27 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:294:27 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:289:27 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:288:27 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:282:25 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:282:25 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:281:25 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:281:25 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:274:27 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:274:27 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:273:27 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:273:27 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:268:27 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:267:27 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:256:6 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:250:27 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:249:27 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:243:25 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:243:25 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:242:25 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:242:25 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:237:25 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:236:25 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:230:6 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:230:6 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:228:19 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:228:19 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:227:19 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:227:19 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:218:4 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:211:6 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:211:6 Styles do not match layer selector #landcover.\n",
      "Warning: Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:195:35 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:195:35 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:194:30 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:194:30 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:193:33 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:193:33 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:192:30 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:192:30 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:189:27 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:188:27 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:180:27 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:180:27 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:179:27 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:179:27 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:174:27 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:173:27 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:166:27 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:166:27 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:165:27 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:165:27 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:159:27 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:158:27 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:151:27 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:151:27 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:150:27 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:150:27 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:145:27 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:144:27 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:137:27 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:137:27 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:136:27 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:136:27 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:131:27 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:130:27 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:124:25 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:123:25 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:120:6 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:120:6 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:112:27 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:111:27 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:108:8 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:108:8 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:99:25 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:99:25 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:98:25 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:98:25 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:95:6 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:95:6 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:86:25 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:86:25 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:85:25 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:85:25 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:82:6 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:82:6 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:693:4 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:683:4 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:672:6 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:665:4 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:658:6 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:650:4 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:641:4 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:641:4 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:633:4 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:628:4 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:628:4 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:617:6 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:617:6 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:601:4 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:591:6 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:591:6 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:588:6 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:588:6 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:583:6 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:561:6 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:550:6 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:542:4 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:536:4 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:530:4 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:524:4 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:518:4 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:512:4 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:500:6 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:487:4 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:481:4 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:474:6 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:466:6 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:449:4 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:433:4 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:433:4 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:421:4 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:412:6 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:412:6 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:409:6 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:409:6 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:398:6 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:398:6 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:395:6 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:395:6 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:385:4 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:385:4 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:370:4 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:354:6 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:345:4 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:331:6 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:316:4 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:309:6 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:298:6 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:298:6 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:292:6 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:292:6 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:287:6 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:280:4 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:280:4 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:271:6 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:271:6 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:266:6 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:248:6 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:241:4 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:241:4 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:226:4 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:208:4 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:208:4 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:187:6 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:177:6 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:177:6 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:172:6 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:162:6 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:162:6 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:157:6 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:148:6 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:148:6 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:143:6 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:134:6 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:134:6 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:129:6 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:117:4 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:106:6 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:93:4 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:93:4 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:80:4 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:80:4 Styles do not match layer selector #landcover.\n",
      "Warning: style/landcover.mss:407:4 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:393:4 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/landcover.mss:74:44 Styles do not match layer selector #landcover-low-zoom.\n",
      "Warning: style/water.mss:305:8 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/water.mss:315:8 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/water.mss:310:8 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/water.mss:299:6 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/water-features.mss:155:8 Styles do not match layer selector #text-line.\n",
      "Warning: style/water-features.mss:137:8 Styles do not match layer selector #text-point.\n",
      "Warning: style/water-features.mss:132:8 Styles do not match layer selector #text-line.\n",
      "Warning: style/water-features.mss:91:21 Styles do not match layer selector #piers-line.\n",
      "Warning: style/water-features.mss:90:21 Styles do not match layer selector #piers-line.\n",
      "Warning: style/water-features.mss:78:21 Styles do not match layer selector #piers-line.\n",
      "Warning: style/water-features.mss:77:21 Styles do not match layer selector #piers-line.\n",
      "Warning: style/water-features.mss:76:21 Styles do not match layer selector #piers-line.\n",
      "Warning: style/water-features.mss:60:21 Styles do not match layer selector #water-barriers-point.\n",
      "Warning: style/water-features.mss:45:21 Styles do not match layer selector #water-barriers-point.\n",
      "Warning: style/water-features.mss:28:21 Styles do not match layer selector #water-barriers-point.\n",
      "Warning: style/water-features.mss:148:6 Styles do not match layer selector #text-point.\n",
      "Warning: style/water-features.mss:148:6 Styles do not match layer selector #text-line.\n",
      "Warning: style/water-features.mss:125:6 Styles do not match layer selector #text-point.\n",
      "Warning: style/water-features.mss:125:6 Styles do not match layer selector #text-line.\n",
      "Warning: style/water-features.mss:88:6 Styles do not match layer selector #piers-line.\n",
      "Warning: style/water-features.mss:85:6 Styles do not match layer selector #piers-poly.\n",
      "Warning: style/water-features.mss:73:6 Styles do not match layer selector #piers-line.\n",
      "Warning: style/water-features.mss:70:6 Styles do not match layer selector #piers-poly.\n",
      "Warning: style/water-features.mss:57:6 Styles do not match layer selector #water-barriers-point.\n",
      "Warning: style/water-features.mss:53:6 Styles do not match layer selector #water-barriers-line.\n",
      "Warning: style/water-features.mss:41:6 Styles do not match layer selector #water-barriers-point.\n",
      "Warning: style/water-features.mss:36:6 Styles do not match layer selector #water-barriers-line.\n",
      "Warning: style/water-features.mss:24:6 Styles do not match layer selector #water-barriers-point.\n",
      "Warning: style/water-features.mss:18:6 Styles do not match layer selector #water-barriers-line.\n",
      "Warning: style/water-features.mss:11:6 Styles do not match layer selector #water-barriers-poly.\n",
      "Warning: style/roads.mss:1868:12 Styles do not match layer selector #roads-fill.\n",
      "Warning: style/roads.mss:1865:12 Styles do not match layer selector #roads-fill.\n",
      "Warning: style/roads.mss:1862:12 Styles do not match layer selector #roads-fill.\n",
      "Warning: style/roads.mss:1949:12 Styles do not match layer selector #roads-fill.\n",
      "Warning: style/roads.mss:1946:12 Styles do not match layer selector #roads-fill.\n",
      "Warning: style/roads.mss:1943:12 Styles do not match layer selector #roads-fill.\n",
      "Warning: style/roads.mss:1856:10 Styles do not match layer selector #roads-fill.\n",
      "Warning: style/roads.mss:2139:12 Styles do not match layer selector #tunnels.\n",
      "Warning: style/roads.mss:2135:12 Styles do not match layer selector #tunnels.\n",
      "Warning: style/roads.mss:2120:14 Styles do not match layer selector #roads-fill.\n",
      "Warning: style/roads.mss:2120:14 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:2110:14 Styles do not match layer selector #roads-fill.\n",
      "Warning: style/roads.mss:2110:14 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:2107:14 Styles do not match layer selector #roads-fill.\n",
      "Warning: style/roads.mss:2107:14 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:2103:14 Styles do not match layer selector #roads-fill.\n",
      "Warning: style/roads.mss:2103:14 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:2028:12 Styles do not match layer selector #roads-fill.\n",
      "Warning: style/roads.mss:2025:12 Styles do not match layer selector #roads-fill.\n",
      "Warning: style/roads.mss:1937:10 Styles do not match layer selector #roads-fill.\n",
      "Warning: style/roads.mss:1833:10 Styles do not match layer selector #roads-fill.\n",
      "Warning: style/roads.mss:1754:27 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1753:27 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1752:27 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1751:27 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1747:27 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1746:27 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1745:27 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1744:27 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1743:27 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1610:29 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1609:29 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1608:29 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1607:29 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1606:29 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1560:29 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1559:29 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1558:29 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1557:29 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1556:29 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1553:27 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1552:27 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1551:27 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1550:27 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1549:27 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1548:27 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1547:27 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1506:29 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1505:29 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1504:29 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1503:29 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1502:29 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1499:27 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1498:27 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1497:27 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1496:27 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1495:27 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1457:29 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1456:29 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1455:29 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1454:29 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1453:29 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1450:27 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1449:27 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1448:27 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1447:27 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1446:27 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1406:29 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1405:29 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1404:29 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1403:29 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1402:29 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1401:29 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1398:27 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1397:27 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1396:27 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1395:27 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1394:27 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1393:27 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:912:25 Styles do not match layer selector #tunnels.\n",
      "Warning: style/roads.mss:911:25 Styles do not match layer selector #tunnels.\n",
      "Warning: style/roads.mss:910:25 Styles do not match layer selector #tunnels.\n",
      "Warning: style/roads.mss:909:25 Styles do not match layer selector #tunnels.\n",
      "Warning: style/roads.mss:897:25 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:896:25 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:895:25 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:894:25 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:704:25 Styles do not match layer selector #tunnels.\n",
      "Warning: style/roads.mss:703:25 Styles do not match layer selector #tunnels.\n",
      "Warning: style/roads.mss:702:25 Styles do not match layer selector #tunnels.\n",
      "Warning: style/roads.mss:701:25 Styles do not match layer selector #tunnels.\n",
      "Warning: style/roads.mss:692:25 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:691:25 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:690:25 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:689:25 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:2131:12 Styles do not match layer selector #tunnels.\n",
      "Warning: style/roads.mss:2115:12 Styles do not match layer selector #roads-fill.\n",
      "Warning: style/roads.mss:2115:12 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:2098:12 Styles do not match layer selector #roads-fill.\n",
      "Warning: style/roads.mss:2098:12 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:2018:10 Styles do not match layer selector #roads-fill.\n",
      "Warning: style/roads.mss:1845:10 Styles do not match layer selector #tunnels.\n",
      "Warning: style/roads.mss:1814:10 Styles do not match layer selector #roads-fill.\n",
      "Warning: style/roads.mss:1750:12 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1742:12 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1605:14 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1603:27 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1602:27 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1601:27 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1600:27 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1599:27 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1598:27 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1597:27 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1555:14 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1546:12 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1543:12 Styles do not match layer selector #tunnels.\n",
      "Warning: style/roads.mss:1501:14 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1494:12 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1491:12 Styles do not match layer selector #tunnels.\n",
      "Warning: style/roads.mss:1452:14 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1445:12 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1442:12 Styles do not match layer selector #tunnels.\n",
      "Warning: style/roads.mss:1400:14 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1392:12 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1389:12 Styles do not match layer selector #tunnels.\n",
      "Warning: style/roads.mss:1015:14 Styles do not match layer selector #tunnels.\n",
      "Warning: style/roads.mss:1012:14 Styles do not match layer selector #tunnels.\n",
      "Warning: style/roads.mss:941:25 Styles do not match layer selector #tunnels.\n",
      "Warning: style/roads.mss:940:25 Styles do not match layer selector #tunnels.\n",
      "Warning: style/roads.mss:939:25 Styles do not match layer selector #tunnels.\n",
      "Warning: style/roads.mss:938:25 Styles do not match layer selector #tunnels.\n",
      "Warning: style/roads.mss:926:25 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:925:25 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:924:25 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:923:25 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:905:10 Styles do not match layer selector #tunnels.\n",
      "Warning: style/roads.mss:893:10 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:883:25 Styles do not match layer selector #tunnels.\n",
      "Warning: style/roads.mss:871:25 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:782:14 Styles do not match layer selector #tunnels.\n",
      "Warning: style/roads.mss:779:14 Styles do not match layer selector #tunnels.\n",
      "Warning: style/roads.mss:732:25 Styles do not match layer selector #tunnels.\n",
      "Warning: style/roads.mss:731:25 Styles do not match layer selector #tunnels.\n",
      "Warning: style/roads.mss:730:25 Styles do not match layer selector #tunnels.\n",
      "Warning: style/roads.mss:729:25 Styles do not match layer selector #tunnels.\n",
      "Warning: style/roads.mss:720:25 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:719:25 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:718:25 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:717:25 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:700:10 Styles do not match layer selector #tunnels.\n",
      "Warning: style/roads.mss:688:10 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:676:25 Styles do not match layer selector #tunnels.\n",
      "Warning: style/roads.mss:667:25 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:2202:10 Styles do not match layer selector #tunnels.\n",
      "Warning: style/roads.mss:2126:10 Styles do not match layer selector #tunnels.\n",
      "Warning: style/roads.mss:2094:10 Styles do not match layer selector #roads-fill.\n",
      "Warning: style/roads.mss:2094:10 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:2089:10 Styles do not match layer selector #tunnels.\n",
      "Warning: style/roads.mss:1775:25 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1774:25 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1773:25 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1772:25 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1771:25 Styles do not match layer selector #bridges.\n",
      "Warning: Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1738:10 Styles do not match layer selector #tunnels.\n",
      "Warning: style/roads.mss:1707:25 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1706:25 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1705:25 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1704:25 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1679:25 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1678:25 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1677:25 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1676:25 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1675:25 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1674:25 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1650:25 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1649:25 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1648:25 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1647:25 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1646:25 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1645:25 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1596:12 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1593:12 Styles do not match layer selector #tunnels.\n",
      "Warning: style/roads.mss:1061:12 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1010:12 Styles do not match layer selector #tunnels.\n",
      "Warning: style/roads.mss:1006:12 Styles do not match layer selector #tunnels.\n",
      "Warning: style/roads.mss:1003:12 Styles do not match layer selector #tunnels.\n",
      "Warning: style/roads.mss:979:12 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:976:12 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:963:25 Styles do not match layer selector #tunnels.\n",
      "Warning: style/roads.mss:951:25 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:934:10 Styles do not match layer selector #tunnels.\n",
      "Warning: style/roads.mss:922:10 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:879:10 Styles do not match layer selector #tunnels.\n",
      "Warning: style/roads.mss:870:10 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:796:12 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:777:12 Styles do not match layer selector #tunnels.\n",
      "Warning: style/roads.mss:774:12 Styles do not match layer selector #tunnels.\n",
      "Warning: style/roads.mss:771:12 Styles do not match layer selector #tunnels.\n",
      "Warning: style/roads.mss:749:12 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:746:12 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:728:10 Styles do not match layer selector #tunnels.\n",
      "Warning: style/roads.mss:716:10 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:675:10 Styles do not match layer selector #tunnels.\n",
      "Warning: style/roads.mss:666:10 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:654:25 Styles do not match layer selector #tunnels.\n",
      "Warning: style/roads.mss:645:25 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:2217:10 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:2212:10 Styles do not match layer selector #tunnels.\n",
      "Warning: style/roads.mss:2156:10 Styles do not match layer selector #tunnels.\n",
      "Warning: style/roads.mss:1770:10 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1711:10 Styles do not match layer selector #tunnels.\n",
      "Warning: style/roads.mss:1703:10 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1700:10 Styles do not match layer selector #roads-fill.\n",
      "Warning: style/roads.mss:1673:10 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1670:10 Styles do not match layer selector #tunnels.\n",
      "Warning: style/roads.mss:1667:10 Styles do not match layer selector #roads-fill.\n",
      "Warning: style/roads.mss:1667:10 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1644:10 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1641:10 Styles do not match layer selector #tunnels.\n",
      "Warning: style/roads.mss:1638:10 Styles do not match layer selector #roads-fill.\n",
      "Warning: style/roads.mss:1638:10 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1072:10 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1059:10 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1048:10 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:997:10 Styles do not match layer selector #tunnels.\n",
      "Warning: style/roads.mss:990:12 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:987:12 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:972:10 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:959:10 Styles do not match layer selector #tunnels.\n",
      "Warning: style/roads.mss:950:10 Styles do not match layer selector #bridges.\n",
      "Warning: Styles do not match layer selector #tunnels.\n",
      "Warning: style/roads.mss:855:10 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:832:10 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:808:10 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:794:10 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:767:10 Styles do not match layer selector #tunnels.\n",
      "Warning: style/roads.mss:760:12 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:757:12 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:742:10 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:653:10 Styles do not match layer selector #tunnels.\n",
      "Warning: style/roads.mss:644:10 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:633:12 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:582:10 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:579:10 Styles do not match layer selector #tunnels.\n",
      "Warning: style/roads.mss:575:10 Styles do not match layer selector #roads-casing.\n",
      "Warning: style/roads.mss:526:12 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:499:12 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:464:12 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:428:25 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:394:25 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:360:25 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1084:10 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1035:10 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1026:8 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:983:10 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:842:10 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:820:10 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:753:10 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:629:10 Styles do not match layer selector #tunnels.\n",
      "Warning: style/roads.mss:625:10 Styles do not match layer selector #roads-casing.\n",
      "Warning: style/roads.mss:605:10 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:602:10 Styles do not match layer selector #tunnels.\n",
      "Warning: style/roads.mss:598:10 Styles do not match layer selector #roads-casing.\n",
      "Warning: style/roads.mss:549:10 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:546:10 Styles do not match layer selector #tunnels.\n",
      "Warning: style/roads.mss:542:10 Styles do not match layer selector #roads-casing.\n",
      "Warning: style/roads.mss:522:10 Styles do not match layer selector #tunnels.\n",
      "Warning: style/roads.mss:518:10 Styles do not match layer selector #roads-casing.\n",
      "Warning: style/roads.mss:495:10 Styles do not match layer selector #tunnels.\n",
      "Warning: style/roads.mss:491:10 Styles do not match layer selector #roads-casing.\n",
      "Warning: style/roads.mss:460:10 Styles do not match layer selector #tunnels.\n",
      "Warning: style/roads.mss:456:10 Styles do not match layer selector #roads-casing.\n",
      "Warning: style/roads.mss:427:10 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:424:10 Styles do not match layer selector #tunnels.\n",
      "Warning: style/roads.mss:420:10 Styles do not match layer selector #roads-casing.\n",
      "Warning: style/roads.mss:393:10 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:390:10 Styles do not match layer selector #tunnels.\n",
      "Warning: style/roads.mss:386:10 Styles do not match layer selector #roads-casing.\n",
      "Warning: style/roads.mss:359:10 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:356:10 Styles do not match layer selector #tunnels.\n",
      "Warning: style/roads.mss:352:10 Styles do not match layer selector #roads-casing.\n",
      "Warning: style/roads.mss:2089:10 Styles do not match layer selector .roads_low_zoom.\n",
      "Warning: style/roads.mss:1926:12 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1923:12 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1920:12 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1909:12 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1906:12 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1902:12 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1891:12 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1888:12 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1885:12 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1881:12 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:2007:12 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:2004:12 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:2001:12 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1990:12 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1987:12 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1983:12 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1972:12 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1969:12 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1966:12 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1962:12 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1914:28 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1913:10 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1896:28 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1895:10 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1878:10 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1317:14 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1313:14 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1304:14 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1300:14 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1295:14 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1995:28 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1994:10 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1977:28 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1976:10 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1959:10 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1309:12 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1292:12 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1212:12 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1208:12 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1158:23 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1157:23 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1156:22 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1155:22 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1152:23 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1151:23 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1150:22 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1149:22 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1148:22 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1147:22 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1144:23 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1143:23 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1142:22 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1141:22 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1140:22 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1139:22 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:2198:12 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:2193:12 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:2188:12 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:2182:12 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:2074:12 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:2071:12 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:2068:12 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:2065:12 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:2062:12 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1872:26 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1871:8 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1843:23 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1733:25 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1732:25 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1731:25 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1730:25 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1726:25 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1725:25 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1724:25 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1723:25 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1722:25 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1590:27 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1589:27 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1588:27 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1587:27 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1586:27 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1540:27 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1539:27 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1538:27 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1537:27 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1536:27 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1533:25 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1532:25 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1531:25 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1530:25 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1529:25 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1528:25 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1527:25 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1488:27 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1487:27 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1486:27 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1485:27 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1484:27 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1481:25 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1480:25 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1479:25 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1478:25 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1477:25 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1439:27 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1438:27 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1437:27 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1436:27 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1435:27 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1432:25 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1431:25 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1430:25 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1429:25 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1428:25 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1386:27 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1385:27 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1384:27 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1383:27 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1382:27 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1381:27 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1378:25 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1377:25 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1376:25 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1375:25 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1374:25 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1373:25 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1341:10 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1337:10 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1332:10 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1288:10 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1280:10 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1275:10 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1271:10 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1260:10 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1255:10 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1251:10 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1240:10 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1235:10 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1231:10 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1217:10 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1204:10 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1199:10 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1193:43 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1191:44 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1189:42 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1187:40 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1185:43 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:2186:10 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:2180:10 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:2177:10 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:2086:23 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:2086:23 Styles do not match layer selector #roads-fill.\n",
      "Warning: style/roads.mss:2086:23 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:2086:23 Styles do not match layer selector #tunnels.\n",
      "Warning: style/roads.mss:2085:22 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:2085:22 Styles do not match layer selector #roads-fill.\n",
      "Warning: style/roads.mss:2085:22 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:2085:22 Styles do not match layer selector #tunnels.\n",
      "Warning: style/roads.mss:2060:10 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:2056:10 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:2053:10 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:2050:10 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:2047:10 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:2044:10 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1953:26 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1952:8 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1840:26 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1839:8 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1824:23 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1729:10 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1721:10 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1585:12 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1583:25 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1582:25 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1581:25 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1580:25 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1579:25 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1578:25 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1577:25 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1535:12 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1523:10 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1521:23 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1520:23 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1483:12 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1475:10 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1473:23 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1472:23 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1434:12 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1426:10 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1424:23 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1423:23 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1380:12 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1371:10 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1369:23 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1368:23 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1351:8 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1326:8 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1265:8 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1245:8 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1225:8 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1194:8 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1160:6 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:2280:10 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:2277:10 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:2245:10 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:2229:10 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:2174:8 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:2154:23 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:2152:23 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:2151:23 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:2083:8 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:2083:8 Styles do not match layer selector #roads-fill.\n",
      "Warning: style/roads.mss:2083:8 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:2083:8 Styles do not match layer selector #tunnels.\n",
      "Warning: style/roads.mss:2034:26 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:2033:8 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1821:26 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1820:8 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1767:23 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1766:23 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1765:23 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1764:23 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1763:23 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1719:8 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1698:23 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1697:23 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1696:23 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1695:23 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1665:23 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1664:23 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1663:23 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1662:23 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1661:23 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1660:23 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1636:23 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1635:23 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1634:23 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1633:23 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1632:23 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1631:23 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1626:8 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1622:8 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1575:10 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1572:10 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1471:22 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1422:22 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1421:22 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1420:22 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1367:22 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1366:22 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1365:22 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:2310:8 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:2297:8 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:2288:8 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:2272:8 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:2257:8 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:2241:8 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:2225:8 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:2209:8 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:2163:8 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1799:8 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1794:21 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1793:21 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1792:21 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1791:21 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1790:21 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1789:21 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1784:8 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1762:8 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1694:8 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1688:8 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1659:8 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1630:8 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1569:8 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1518:8 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1469:8 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1418:8 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1363:8 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1124:6 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1114:6 Styles do not match layer selector #roads-low-zoom.\n",
      "Warning: style/roads.mss:1105:6 Styles do not match layer selector #roads-fill.\n",
      "Warning: style/roads.mss:1105:6 Styles do not match layer selector #bridges.\n",
      "Warning: style/roads.mss:1105:6 Styles do not match layer selector #tunnels.\n",
      "Warning: style/placenames.mss:193:8 Styles do not match layer selector #placenames-medium.\n",
      "Warning: style/placenames.mss:183:8 Styles do not match layer selector #placenames-medium.\n",
      "Warning: style/placenames.mss:179:8 Styles do not match layer selector #placenames-medium.\n",
      "Warning: style/placenames.mss:83:6 Styles do not match layer selector #state-names.\n",
      "Warning: style/placenames.mss:77:6 Styles do not match layer selector #state-names.\n",
      "Warning: style/placenames.mss:71:6 Styles do not match layer selector #state-names.\n",
      "Warning: style/placenames.mss:65:6 Styles do not match layer selector #state-names.\n",
      "Warning: style/placenames.mss:39:6 Styles do not match layer selector #country-names.\n",
      "Warning: style/placenames.mss:34:6 Styles do not match layer selector #country-names.\n",
      "Warning: style/placenames.mss:28:6 Styles do not match layer selector #country-names.\n",
      "Warning: style/placenames.mss:22:6 Styles do not match layer selector #country-names.\n",
      "Warning: style/amenity-points.mss:2309:10 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2256:8 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:1945:4 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2960:8 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2870:6 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2202:8 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2197:8 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:1921:6 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2733:8 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2728:8 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2403:8 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2192:6 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2038:8 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2841:8 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2836:8 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2816:8 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2811:8 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2791:8 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2786:8 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2770:8 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2760:8 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2755:8 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2723:6 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2607:8 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2604:8 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2458:8 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2453:8 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2439:8 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2323:8 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2319:8 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2314:8 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2307:8 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2303:8 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2299:8 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2295:8 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2292:8 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2289:8 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2286:8 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2279:8 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2271:8 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2266:8 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2263:8 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2252:8 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2247:8 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2244:8 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2241:8 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2236:8 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2232:8 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2228:8 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2223:8 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2220:8 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2213:8 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2124:8 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2119:8 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2011:8 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2006:8 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:1905:4 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:1799:6 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2964:49 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2950:6 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2831:6 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2806:6 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2781:6 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2773:8 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2750:6 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2594:6 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2448:6 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2397:6 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2114:6 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2100:8 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2095:8 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2001:6 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:1986:6 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:1865:33 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:1599:6 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:1594:6 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2934:4 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2915:19 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2914:19 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2913:19 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2651:8 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2499:4 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2480:6 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2436:8 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2393:6 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2389:6 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2342:4 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2090:6 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2082:8 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2077:8 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2073:8 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2070:8 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2062:8 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2057:8 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2028:6 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:1924:6 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:1877:6 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:1860:6 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:1841:38 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:1839:36 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:1819:4 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:1806:4 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:1792:6 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:1773:6 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:1747:33 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:1746:35 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:1745:37 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:1744:39 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:1724:43 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:1723:43 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:1722:43 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:1721:43 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:1720:43 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:1681:8 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:1676:47 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:1657:6 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:1619:6 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:1588:4 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2922:4 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2907:4 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2895:4 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2883:4 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2853:4 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2708:4 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2695:6 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2680:6 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2665:6 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2579:6 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2565:6 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2550:6 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2512:4 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2486:4 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2470:4 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2426:6 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2410:4 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2379:4 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2356:4 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2329:4 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:2052:6 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:1970:4 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:1957:4 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:1848:4 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:1834:4 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:1786:4 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:1762:4 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:1738:4 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:1715:4 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:1699:4 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:1687:4 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:1670:6 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:1647:4 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:1629:4 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/amenity-points.mss:1609:4 Styles do not match layer selector #text-poly-low-zoom.\n",
      "Warning: style/admin.mss:431:6 Styles do not match layer selector #admin-mid-zoom.\n",
      "Warning: style/admin.mss:376:6 Styles do not match layer selector #admin-mid-zoom.\n",
      "Warning: style/admin.mss:352:6 Styles do not match layer selector #admin-mid-zoom.\n",
      "Warning: style/admin.mss:437:4 Styles do not match layer selector #admin-mid-zoom.\n",
      "Warning: style/admin.mss:431:6 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:422:4 Styles do not match layer selector #admin-mid-zoom.\n",
      "Warning: style/admin.mss:415:6 Styles do not match layer selector #admin-mid-zoom.\n",
      "Warning: style/admin.mss:398:6 Styles do not match layer selector #admin-mid-zoom.\n",
      "Warning: style/admin.mss:376:6 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:371:6 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:367:6 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:352:6 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:347:6 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:343:6 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:338:6 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:333:6 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:300:6 Styles do not match layer selector #admin-mid-zoom.\n",
      "Warning: style/admin.mss:296:6 Styles do not match layer selector #admin-mid-zoom.\n",
      "Warning: style/admin.mss:273:6 Styles do not match layer selector #admin-mid-zoom.\n",
      "Warning: style/admin.mss:269:6 Styles do not match layer selector #admin-mid-zoom.\n",
      "Warning: style/admin.mss:223:19 Styles do not match layer selector #admin-mid-zoom.\n",
      "Warning: style/admin.mss:222:19 Styles do not match layer selector #admin-mid-zoom.\n",
      "Warning: style/admin.mss:206:6 Styles do not match layer selector #admin-mid-zoom.\n",
      "Warning: style/admin.mss:202:6 Styles do not match layer selector #admin-mid-zoom.\n",
      "Warning: style/admin.mss:178:6 Styles do not match layer selector #admin-mid-zoom.\n",
      "Warning: style/admin.mss:174:6 Styles do not match layer selector #admin-mid-zoom.\n",
      "Warning: style/admin.mss:129:19 Styles do not match layer selector #admin-mid-zoom.\n",
      "Warning: style/admin.mss:128:19 Styles do not match layer selector #admin-mid-zoom.\n",
      "Warning: style/admin.mss:112:6 Styles do not match layer selector #admin-mid-zoom.\n",
      "Warning: style/admin.mss:108:6 Styles do not match layer selector #admin-mid-zoom.\n",
      "Warning: style/admin.mss:76:6 Styles do not match layer selector #admin-mid-zoom.\n",
      "Warning: style/admin.mss:72:6 Styles do not match layer selector #admin-mid-zoom.\n",
      "Warning: style/admin.mss:27:19 Styles do not match layer selector #admin-mid-zoom.\n",
      "Warning: style/admin.mss:26:19 Styles do not match layer selector #admin-mid-zoom.\n",
      "Warning: style/admin.mss:437:4 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:422:4 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:415:6 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:405:6 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:398:6 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:393:6 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:383:6 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:358:4 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:324:4 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:300:6 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:296:6 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:291:6 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:287:6 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:279:6 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:273:6 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:269:6 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:265:6 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:261:6 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:256:6 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:252:6 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:248:6 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:223:19 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:222:19 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:221:19 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:220:19 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:219:19 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:218:18 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:214:6 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:206:6 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:202:6 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:197:6 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:193:6 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:184:6 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:178:6 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:174:6 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:170:6 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:166:6 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:161:6 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:157:6 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:153:6 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:129:19 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:128:19 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:127:19 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:126:19 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:125:19 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:124:18 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:120:6 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:112:6 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:108:6 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:103:6 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:99:6 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:94:6 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:90:6 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:82:6 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:76:6 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:72:6 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:68:6 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:64:6 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:59:6 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:55:6 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:51:6 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:27:19 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:26:19 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:25:19 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:24:19 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:23:19 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:22:18 Styles do not match layer selector #admin-low-zoom.\n",
      "Warning: style/admin.mss:18:6 Styles do not match layer selector #admin-low-zoom.\n",
      "2024-12-19 19:59:34  osm2pgsql version 1.4.1\n",
      "2024-12-19 19:59:34  Database version: 15.10 (Debian 15.10-1.pgdg110+1)\n",
      "2024-12-19 19:59:34  PostGIS version: 3.5\n",
      "2024-12-19 19:59:34  Node-cache: cache=800MB, maxblocks=12800*65536, allocation method=3\n",
      "2024-12-19 19:59:34  Setting up table 'planet_osm_point'\n",
      "2024-12-19 19:59:34  Setting up table 'planet_osm_line'\n",
      "2024-12-19 19:59:34  Setting up table 'planet_osm_polygon'\n",
      "2024-12-19 19:59:34  Setting up table 'planet_osm_roads'\n",
      "2024-12-19 19:59:34  Reading input files done in 0s.                          \n",
      "2024-12-19 19:59:34    Processed 59 nodes in 0s - 59/s\n",
      "2024-12-19 19:59:34    Processed 14 ways in 0s - 14/s\n",
      "2024-12-19 19:59:34    Processed 0 relations in 0s - 0/s\n",
      "2024-12-19 19:59:34  node cache: stored: 59(100.00%), storage efficiency: 0.71% (dense blocks: 1, sparse nodes: 30), hit rate: 100.00%\n",
      "2024-12-19 19:59:34  Clustering table 'planet_osm_point' by geometry...\n",
      "2024-12-19 19:59:34  Clustering table 'planet_osm_line' by geometry...\n",
      "2024-12-19 19:59:34  Clustering table 'planet_osm_polygon' by geometry...\n",
      "2024-12-19 19:59:34  Clustering table 'planet_osm_roads' by geometry...\n",
      "2024-12-19 19:59:34  Creating geometry index on table 'planet_osm_line'...\n",
      "2024-12-19 19:59:34  Creating geometry index on table 'planet_osm_roads'...\n",
      "2024-12-19 19:59:34  Creating geometry index on table 'planet_osm_polygon'...\n",
      "2024-12-19 19:59:34  Creating geometry index on table 'planet_osm_point'...\n",
      "2024-12-19 19:59:34  Analyzing table 'planet_osm_roads'...\n",
      "2024-12-19 19:59:34  Analyzing table 'planet_osm_point'...\n",
      "2024-12-19 19:59:34  Analyzing table 'planet_osm_line'...\n",
      "2024-12-19 19:59:34  Analyzing table 'planet_osm_polygon'...\n",
      "2024-12-19 19:59:34  All postprocessing on table 'planet_osm_polygon' done in 0s.\n",
      "2024-12-19 19:59:34  All postprocessing on table 'planet_osm_point' done in 0s.\n",
      "2024-12-19 19:59:34  All postprocessing on table 'planet_osm_line' done in 0s.\n",
      "2024-12-19 19:59:34  All postprocessing on table 'planet_osm_roads' done in 0s.\n",
      "2024-12-19 19:59:34  osm2pgsql took 0s overall.\n",
      "CREATE INDEX\n",
      "CREATE INDEX\n",
      "CREATE INDEX\n",
      "CREATE INDEX\n",
      "CREATE INDEX\n",
      "CREATE INDEX\n",
      "CREATE INDEX\n",
      "CREATE INDEX\n",
      "CREATE INDEX\n",
      "CREATE INDEX\n",
      "CREATE INDEX\n",
      "CREATE INDEX\n",
      "CREATE INDEX\n",
      "CREATE INDEX\n",
      "CREATE INDEX\n",
      "Generating zoom 15 level\n",
      "Mapnik LOG> 2024-12-19 19:59:34: warning: unable to find face-name 'Noto Sans Syriac Eastern Regular' in FontSet 'fontset-0'\n",
      "Mapnik LOG> 2024-12-19 19:59:34: warning: unable to find face-name 'Noto Sans Tibetan Regular' in FontSet 'fontset-0'\n",
      "Mapnik LOG> 2024-12-19 19:59:34: warning: unable to find face-name 'Noto Emoji Regular' in FontSet 'fontset-0'\n",
      "Mapnik LOG> 2024-12-19 19:59:34: warning: unable to find face-name 'Unifont Medium' in FontSet 'fontset-0'\n",
      "Mapnik LOG> 2024-12-19 19:59:34: warning: unable to find face-name 'unifont Medium' in FontSet 'fontset-0'\n",
      "Mapnik LOG> 2024-12-19 19:59:34: warning: unable to find face-name 'Unifont Upper Medium' in FontSet 'fontset-0'\n",
      "Mapnik LOG> 2024-12-19 19:59:34: warning: unable to find face-name 'Noto Sans Syriac Eastern Regular' in FontSet 'fontset-1'\n",
      "Mapnik LOG> 2024-12-19 19:59:34: warning: unable to find face-name 'Noto Sans Tibetan Regular' in FontSet 'fontset-1'\n",
      "Mapnik LOG> 2024-12-19 19:59:34: warning: unable to find face-name 'Noto Emoji Regular' in FontSet 'fontset-1'\n",
      "Mapnik LOG> 2024-12-19 19:59:34: warning: unable to find face-name 'Unifont Medium' in FontSet 'fontset-1'\n",
      "Mapnik LOG> 2024-12-19 19:59:34: warning: unable to find face-name 'unifont Medium' in FontSet 'fontset-1'\n",
      "Mapnik LOG> 2024-12-19 19:59:34: warning: unable to find face-name 'Unifont Upper Medium' in FontSet 'fontset-1'\n",
      "Mapnik LOG> 2024-12-19 19:59:34: warning: unable to find face-name 'Noto Sans Tibetan Bold' in FontSet 'fontset-2'\n",
      "Mapnik LOG> 2024-12-19 19:59:34: warning: unable to find face-name 'Noto Sans Syriac Eastern Regular' in FontSet 'fontset-2'\n",
      "Mapnik LOG> 2024-12-19 19:59:34: warning: unable to find face-name 'Noto Sans Tibetan Regular' in FontSet 'fontset-2'\n",
      "Mapnik LOG> 2024-12-19 19:59:34: warning: unable to find face-name 'Noto Emoji Regular' in FontSet 'fontset-2'\n",
      "Mapnik LOG> 2024-12-19 19:59:34: warning: unable to find face-name 'Unifont Medium' in FontSet 'fontset-2'\n",
      "Mapnik LOG> 2024-12-19 19:59:34: warning: unable to find face-name 'unifont Medium' in FontSet 'fontset-2'\n",
      "Mapnik LOG> 2024-12-19 19:59:34: warning: unable to find face-name 'Unifont Upper Medium' in FontSet 'fontset-2'\n",
      "Generating zoom 16 level\n",
      "Mapnik LOG> 2024-12-19 19:59:36: warning: unable to find face-name 'Noto Sans Syriac Eastern Regular' in FontSet 'fontset-0'\n",
      "Mapnik LOG> 2024-12-19 19:59:36: warning: unable to find face-name 'Noto Sans Tibetan Regular' in FontSet 'fontset-0'\n",
      "Mapnik LOG> 2024-12-19 19:59:36: warning: unable to find face-name 'Noto Emoji Regular' in FontSet 'fontset-0'\n",
      "Mapnik LOG> 2024-12-19 19:59:36: warning: unable to find face-name 'Unifont Medium' in FontSet 'fontset-0'\n",
      "Mapnik LOG> 2024-12-19 19:59:36: warning: unable to find face-name 'unifont Medium' in FontSet 'fontset-0'\n",
      "Mapnik LOG> 2024-12-19 19:59:36: warning: unable to find face-name 'Unifont Upper Medium' in FontSet 'fontset-0'\n",
      "Mapnik LOG> 2024-12-19 19:59:36: warning: unable to find face-name 'Noto Sans Syriac Eastern Regular' in FontSet 'fontset-1'\n",
      "Mapnik LOG> 2024-12-19 19:59:36: warning: unable to find face-name 'Noto Sans Tibetan Regular' in FontSet 'fontset-1'\n",
      "Mapnik LOG> 2024-12-19 19:59:36: warning: unable to find face-name 'Noto Emoji Regular' in FontSet 'fontset-1'\n",
      "Mapnik LOG> 2024-12-19 19:59:36: warning: unable to find face-name 'Unifont Medium' in FontSet 'fontset-1'\n",
      "Mapnik LOG> 2024-12-19 19:59:36: warning: unable to find face-name 'unifont Medium' in FontSet 'fontset-1'\n",
      "Mapnik LOG> 2024-12-19 19:59:36: warning: unable to find face-name 'Unifont Upper Medium' in FontSet 'fontset-1'\n",
      "Mapnik LOG> 2024-12-19 19:59:36: warning: unable to find face-name 'Noto Sans Tibetan Bold' in FontSet 'fontset-2'\n",
      "Mapnik LOG> 2024-12-19 19:59:36: warning: unable to find face-name 'Noto Sans Syriac Eastern Regular' in FontSet 'fontset-2'\n",
      "Mapnik LOG> 2024-12-19 19:59:36: warning: unable to find face-name 'Noto Sans Tibetan Regular' in FontSet 'fontset-2'\n",
      "Mapnik LOG> 2024-12-19 19:59:36: warning: unable to find face-name 'Noto Emoji Regular' in FontSet 'fontset-2'\n",
      "Mapnik LOG> 2024-12-19 19:59:36: warning: unable to find face-name 'Unifont Medium' in FontSet 'fontset-2'\n",
      "Mapnik LOG> 2024-12-19 19:59:36: warning: unable to find face-name 'unifont Medium' in FontSet 'fontset-2'\n",
      "Mapnik LOG> 2024-12-19 19:59:36: warning: unable to find face-name 'Unifont Upper Medium' in FontSet 'fontset-2'\n",
      "2024-12-19 19:59:37  osm2pgsql version 1.4.1\n",
      "2024-12-19 19:59:37  Database version: 15.10 (Debian 15.10-1.pgdg110+1)\n",
      "2024-12-19 19:59:37  PostGIS version: 3.5\n",
      "2024-12-19 19:59:37  Node-cache: cache=800MB, maxblocks=12800*65536, allocation method=3\n",
      "2024-12-19 19:59:37  Setting up table 'planet_osm_point'\n",
      "2024-12-19 19:59:37  Setting up table 'planet_osm_line'\n",
      "2024-12-19 19:59:37  Setting up table 'planet_osm_polygon'\n",
      "2024-12-19 19:59:37  Setting up table 'planet_osm_roads'\n",
      "2024-12-19 19:59:37  Reading input files done in 0s.                          \n",
      "2024-12-19 19:59:37    Processed 45810 nodes in 0s - 46k/s\n",
      "2024-12-19 19:59:37    Processed 7176 ways in 0s - 7k/s\n",
      "2024-12-19 19:59:37    Processed 55 relations in 0s - 55/s\n",
      "2024-12-19 19:59:37  node cache: stored: 45810(100.00%), storage efficiency: 45.90% (dense blocks: 1, sparse nodes: 45808), hit rate: 100.00%\n",
      "2024-12-19 19:59:37  Clustering table 'planet_osm_point' by geometry...\n",
      "2024-12-19 19:59:37  Clustering table 'planet_osm_line' by geometry...\n",
      "2024-12-19 19:59:37  Clustering table 'planet_osm_polygon' by geometry...\n",
      "2024-12-19 19:59:37  Clustering table 'planet_osm_roads' by geometry...\n",
      "2024-12-19 19:59:37  Creating geometry index on table 'planet_osm_roads'...\n",
      "2024-12-19 19:59:37  Analyzing table 'planet_osm_roads'...\n",
      "2024-12-19 19:59:37  All postprocessing on table 'planet_osm_roads' done in 0s.\n",
      "2024-12-19 19:59:37  Creating geometry index on table 'planet_osm_line'...\n",
      "2024-12-19 19:59:37  Creating geometry index on table 'planet_osm_point'...\n",
      "2024-12-19 19:59:37  Analyzing table 'planet_osm_line'...\n",
      "2024-12-19 19:59:37  Analyzing table 'planet_osm_point'...\n",
      "2024-12-19 19:59:37  All postprocessing on table 'planet_osm_point' done in 0s.\n",
      "2024-12-19 19:59:37  All postprocessing on table 'planet_osm_line' done in 0s.\n",
      "2024-12-19 19:59:37  Creating geometry index on table 'planet_osm_polygon'...\n",
      "2024-12-19 19:59:37  Analyzing table 'planet_osm_polygon'...\n",
      "2024-12-19 19:59:37  All postprocessing on table 'planet_osm_polygon' done in 0s.\n",
      "2024-12-19 19:59:37  osm2pgsql took 0s overall.\n",
      "CREATE INDEX\n",
      "CREATE INDEX\n",
      "CREATE INDEX\n",
      "CREATE INDEX\n",
      "CREATE INDEX\n",
      "CREATE INDEX\n",
      "CREATE INDEX\n",
      "CREATE INDEX\n",
      "CREATE INDEX\n",
      "CREATE INDEX\n",
      "CREATE INDEX\n",
      "CREATE INDEX\n",
      "CREATE INDEX\n",
      "CREATE INDEX\n",
      "CREATE INDEX\n",
      "Generating zoom 15 level\n",
      "Mapnik LOG> 2024-12-19 19:59:38: warning: unable to find face-name 'Noto Sans Syriac Eastern Regular' in FontSet 'fontset-0'\n",
      "Mapnik LOG> 2024-12-19 19:59:38: warning: unable to find face-name 'Noto Sans Tibetan Regular' in FontSet 'fontset-0'\n",
      "Mapnik LOG> 2024-12-19 19:59:38: warning: unable to find face-name 'Noto Emoji Regular' in FontSet 'fontset-0'\n",
      "Mapnik LOG> 2024-12-19 19:59:38: warning: unable to find face-name 'Unifont Medium' in FontSet 'fontset-0'\n",
      "Mapnik LOG> 2024-12-19 19:59:38: warning: unable to find face-name 'unifont Medium' in FontSet 'fontset-0'\n",
      "Mapnik LOG> 2024-12-19 19:59:38: warning: unable to find face-name 'Unifont Upper Medium' in FontSet 'fontset-0'\n",
      "Mapnik LOG> 2024-12-19 19:59:38: warning: unable to find face-name 'Noto Sans Syriac Eastern Regular' in FontSet 'fontset-1'\n",
      "Mapnik LOG> 2024-12-19 19:59:38: warning: unable to find face-name 'Noto Sans Tibetan Regular' in FontSet 'fontset-1'\n",
      "Mapnik LOG> 2024-12-19 19:59:38: warning: unable to find face-name 'Noto Emoji Regular' in FontSet 'fontset-1'\n",
      "Mapnik LOG> 2024-12-19 19:59:38: warning: unable to find face-name 'Unifont Medium' in FontSet 'fontset-1'\n",
      "Mapnik LOG> 2024-12-19 19:59:38: warning: unable to find face-name 'unifont Medium' in FontSet 'fontset-1'\n",
      "Mapnik LOG> 2024-12-19 19:59:38: warning: unable to find face-name 'Unifont Upper Medium' in FontSet 'fontset-1'\n",
      "Mapnik LOG> 2024-12-19 19:59:38: warning: unable to find face-name 'Noto Sans Tibetan Bold' in FontSet 'fontset-2'\n",
      "Mapnik LOG> 2024-12-19 19:59:38: warning: unable to find face-name 'Noto Sans Syriac Eastern Regular' in FontSet 'fontset-2'\n",
      "Mapnik LOG> 2024-12-19 19:59:38: warning: unable to find face-name 'Noto Sans Tibetan Regular' in FontSet 'fontset-2'\n",
      "Mapnik LOG> 2024-12-19 19:59:38: warning: unable to find face-name 'Noto Emoji Regular' in FontSet 'fontset-2'\n",
      "Mapnik LOG> 2024-12-19 19:59:38: warning: unable to find face-name 'Unifont Medium' in FontSet 'fontset-2'\n",
      "Mapnik LOG> 2024-12-19 19:59:38: warning: unable to find face-name 'unifont Medium' in FontSet 'fontset-2'\n",
      "Mapnik LOG> 2024-12-19 19:59:38: warning: unable to find face-name 'Unifont Upper Medium' in FontSet 'fontset-2'\n",
      "Generating zoom 16 level\n",
      "Mapnik LOG> 2024-12-19 19:59:39: warning: unable to find face-name 'Noto Sans Syriac Eastern Regular' in FontSet 'fontset-0'\n",
      "Mapnik LOG> 2024-12-19 19:59:39: warning: unable to find face-name 'Noto Sans Tibetan Regular' in FontSet 'fontset-0'\n",
      "Mapnik LOG> 2024-12-19 19:59:39: warning: unable to find face-name 'Noto Emoji Regular' in FontSet 'fontset-0'\n",
      "Mapnik LOG> 2024-12-19 19:59:39: warning: unable to find face-name 'Unifont Medium' in FontSet 'fontset-0'\n",
      "Mapnik LOG> 2024-12-19 19:59:39: warning: unable to find face-name 'unifont Medium' in FontSet 'fontset-0'\n",
      "Mapnik LOG> 2024-12-19 19:59:39: warning: unable to find face-name 'Unifont Upper Medium' in FontSet 'fontset-0'\n",
      "Mapnik LOG> 2024-12-19 19:59:39: warning: unable to find face-name 'Noto Sans Syriac Eastern Regular' in FontSet 'fontset-1'\n",
      "Mapnik LOG> 2024-12-19 19:59:39: warning: unable to find face-name 'Noto Sans Tibetan Regular' in FontSet 'fontset-1'\n",
      "Mapnik LOG> 2024-12-19 19:59:39: warning: unable to find face-name 'Noto Emoji Regular' in FontSet 'fontset-1'\n",
      "Mapnik LOG> 2024-12-19 19:59:39: warning: unable to find face-name 'Unifont Medium' in FontSet 'fontset-1'\n",
      "Mapnik LOG> 2024-12-19 19:59:39: warning: unable to find face-name 'unifont Medium' in FontSet 'fontset-1'\n",
      "Mapnik LOG> 2024-12-19 19:59:39: warning: unable to find face-name 'Unifont Upper Medium' in FontSet 'fontset-1'\n",
      "Mapnik LOG> 2024-12-19 19:59:39: warning: unable to find face-name 'Noto Sans Tibetan Bold' in FontSet 'fontset-2'\n",
      "Mapnik LOG> 2024-12-19 19:59:39: warning: unable to find face-name 'Noto Sans Syriac Eastern Regular' in FontSet 'fontset-2'\n",
      "Mapnik LOG> 2024-12-19 19:59:39: warning: unable to find face-name 'Noto Sans Tibetan Regular' in FontSet 'fontset-2'\n",
      "Mapnik LOG> 2024-12-19 19:59:39: warning: unable to find face-name 'Noto Emoji Regular' in FontSet 'fontset-2'\n",
      "Mapnik LOG> 2024-12-19 19:59:39: warning: unable to find face-name 'Unifont Medium' in FontSet 'fontset-2'\n",
      "Mapnik LOG> 2024-12-19 19:59:39: warning: unable to find face-name 'unifont Medium' in FontSet 'fontset-2'\n",
      "Mapnik LOG> 2024-12-19 19:59:39: warning: unable to find face-name 'Unifont Upper Medium' in FontSet 'fontset-2'\n",
      "Generating comparison image for zoom 15\n",
      "Generating comparison image for zoom 16\n"
     ]
    }
   ],
   "source": [
    "!./make.sh {filename.value} {start_date.value.isoformat()}Z {end_date.value.isoformat()}Z {bbox.value} {min_zoom_level.value} {max_zoom_level.value} {num_frames.value}"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "1bd7bf97-b569-4001-8e87-7ece096db213",
   "metadata": {},
   "source": [
    "Enjoy mapping and visualizing!\n",
    "\n",
    "If this all worked you should be able to see the output files in the filebrowser/file tree, named `progress.filename.parameters.png/gif.`. \n",
    "\n",
    "If you are running this in the MyBinder, you should download them to your computer to keep them permanently.\n",
    "\n",
    "If you are running this locally, you can do this as well or alternatively move them into the shared \"output\" folder that is accessible from your host computer.  \n",
    "\n"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.2"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
```

Reply:
Perfect! I’ll see if I can combine things 🙏